### PR TITLE
image_spliter

### PR DESCRIPTION
**Pull request by CalebMuhia**

### CODE REVIEW

File: image_spliter.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        return
+
+    for path in paths:
+        image_results = {}
+        results = []
+        tiles = image_slicer.slice(path, parts, save=False)
+        for ind, tile in enumerate(tiles):
+            image = io.BytesIO()
+            filename, ext = os.path.splitext(path)
+            if 'png' in ext.lower():
```
Comment (marcbelmont) [inline image_spliter.py]: getting the extension is not necessary. Always use JPEG

**Comment by marcbelmont** [comment]
rename file "image_splitter.py"

File: image_spliter.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+                if response.status_code == 429:  # Max calls per second reached
+                    time.sleep(1)
+
+            results.append((tile, response.json()))
+
+        image_results["processing_time"] = sum(
+            [result[1]['processing_time'] for result in results])
+
+        for result in results:
```
[the remark this answers is not in the document]
Comment (CalebMuhia) [reply image_spliter.py]: updated

File: image_spliter.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    parts = int(args.parts)
+    camera_id = args.camera_id
+
+    if not args.sdk_url and not args.api_key:
+        raise Exception('api-key is required')
+    if len(paths) == 0:
+        print('File {} does not exist.'.format(args.FILE))
+        return
+
+    for path in paths:
```
[the remark this answers is not in the document]
Comment (CalebMuhia) [reply image_spliter.py]: updated

File: image_spliter.py (hunk 1 above)
Comment (CalebMuhia) [reply image_spliter.py]: updated

**Comment by marcbelmont** [comment]
I have opted for a different approach. The splitting function used in this script would not allow overlap.

**Pull request closed without merging**